src/common/doc/installation.h

```diff
@@ -204,6 +204,8 @@ make doxygen
  * Nerve_GIC/Nerve.cpp</a>
  * \li <a href="_nerve__g_i_c_2_voronoi_g_i_c_8cpp-example.html">
  * Nerve_GIC/VoronoiGIC.cpp</a>
+ * \li <a href="_simplex_tree_2mini_simplex_tree_8cpp-example.html">
+ * Simplex_tree/mini_simplex_tree.cpp</a>
```

Comment:
I wonder about this. The heading says "Having Intel&reg; TBB installed is recommended to parallelize and accelerate some GUDHI computations. The following examples/utilities are using Intel&reg; TBB if installed". This example gets linked with TBB, but that's artificial, it doesn't actually use TBB. So it depends what the goal of this list is. If the goal was to see what may have a dependency on TBB, then we should include it. If the goal is to see what examples benefit from TBB, which seems to be the case, then maybe we should not list it? What do you think?
(the rest of the patch looks fine, thanks)

Reply:
Ok, I see what you mean.
I am ok to remove this part.